python/startup/bootstrap.py

```diff
@@ -1,47 +1,66 @@
 # Copyright (c) 2016 Shotgun Software Inc.
-# 
+#
 # CONFIDENTIAL AND PROPRIETARY
-# 
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your 
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import os
 import sys
+import sgtk
+import uuid
+import imp
+
+logger = sgtk.LogManager.get_logger(__name__)
+
+
+def _get_current_module_path():
+    """
+    Returns the current module's absolute path.
+    """
+    return os.path.dirname(
+        os.path.abspath(sys.modules[_get_current_module_path.__module__].__file__)
```

Comment:
I don't know honestly. That complicated code was already there before. Let me check.